src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IConversionExpression.vb

```diff
@@ -0,0 +1,2116 @@
+        ILocalReferenceExpression: c2List (OperationKind.LocalReferenceExpression, Type: Program.C2()) (Syntax: 'c2List')
+]]>.Value
+
+            Dim expectedDiagnostics = <![CDATA[
+BC36754: 'Program.C2()' cannot be converted to 'IEnumerable(Of Program.C1)' because 'Program.C2' is not derived from 'Program.C1', as required for the 'Out' generic parameter 'T' in 'Interface IEnumerable(Of Out T)'.
+        Dim c1List As IEnumerable(Of C1) = c2List'BIND:"Dim c1List As IEnumerable(Of C1) = c2List"
+                                           ~~~~~~
+]]>.Value
+
+            VerifyOperationTreeAndDiagnosticsForTest(Of LocalDeclarationStatementSyntax)(source, expectedOperationTree, expectedDiagnostics,
+                additionalOperationTreeVerifier:=AddressOf New ExpectedSymbolVerifier().Verify)
+        End Sub
+
+        <Fact()>
+        Public Sub ConversionExpression_Implicit_WideningStructureToInterface()
+            Dim source = <![CDATA[
+Module Program
+    Sub M1()
+        Dim s1 = New S1
+        Dim i1 As I1 = s1'BIND:"Dim i1 As I1 = s1"
+    End Sub
+
+    Interface I1
+    End Interface
+
+    Structure S1
+        Implements I1
+    End Structure
+End Module]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclarationStatement) (Syntax: 'Dim i1 As I1 = s1')
+  IVariableDeclaration (1 variables) (OperationKind.VariableDeclaration) (Syntax: 'i1')
+    Variables: Local_1: i1 As Program.I1
+    Initializer: IConversionExpression (ConversionKind.Basic, Implicit) (OperationKind.ConversionExpression, Type: Program.I1) (Syntax: 's1')
+        ILocalReferenceExpression: s1 (OperationKind.LocalReferenceExpression, Type: Program.S1) (Syntax: 's1')
+]]>.Value
+
+            Dim expectedDiagnostics = String.Empty
+
+            VerifyOperationTreeAndDiagnosticsForTest(Of LocalDeclarationStatementSyntax)(source, expectedOperationTree, expectedDiagnostics)
+        End Sub
+
+        <Fact()>
+        Public Sub ConversionExpression_Implicit_WideningStructureToValueType()
+            Dim source = <![CDATA[
+Option Strict On
+Imports System
+Module Program
+    Sub M1()
+        Dim s1 = New S1
+        Dim v1 As ValueType = s1'BIND:"Dim v1 As ValueType = s1"
+    End Sub
+
+    Structure S1
+    End Structure
+End Module]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclarationStatement) (Syntax: 'Dim v1 As ValueType = s1')
+  IVariableDeclaration (1 variables) (OperationKind.VariableDeclaration) (Syntax: 'v1')
+    Variables: Local_1: v1 As System.ValueType
+    Initializer: IConversionExpression (ConversionKind.Basic, Implicit) (OperationKind.ConversionExpression, Type: System.ValueType) (Syntax: 's1')
+        ILocalReferenceExpression: s1 (OperationKind.LocalReferenceExpres
```

Comment:
Similar answer. I don't think the end result should be different, but since the spec specifically calls out transitive conversions, and I want to make sure the semantic model agrees with the number of implicit conversions we have here (just one from interface to transitively-implementing base), I'm testing it explicitly.